### DISCUSSION TITLE
Does editorjs support emoji?

### DISCUSSION THREAD

**Discussion by wutsi** [codex-team/editor.js#1878] [Q&A]
Im wondering is editorjs support emoji? if yes, is it possible to integrate customer emoji?
Thank you

**Comment by neSpecc** [comment]
What do you mean?

**Comment by wutsi** [comment]
- User can add emojis  into their text will editing it
- If possible, can we integrate a custom set of emojis
Thanks

**Comment by jeromecance** [comment]
I'm also looking after this, in fact, I'm searching a way to always display the inline toolbar so that you can include some customized icons in a paragraph from the inline toolbar without selecting a text.
Any way to do it?

**Comment by semoal** [comment]
> I'm also looking after this, in fact, I'm searching a way to always display the inline toolbar so that you can include some customized icons in a paragraph from the inline toolbar without selecting a text.
> Any way to do it?

Just create a inline toolbar tool. Should be pretty easy though.

**Comment by jameslahm** [comment]
Hi, I just make a custom emoji picker example with editorjs in [here](https://codesandbox.io/s/react-editor-js-forked-yejns?file=/index.js). And you can integrate custom emoji and pick it. Also, it outputs clean data.

**Comment by victorzhrn** [comment]
emoji inline toolbar tool! strongly recommend

**Comment by endlessuniverse** [comment]
Can this be integrated? https://github.com/mervick/emojionearea
Thanks!

**Comment by jeremyVignelles** [comment]
I'd also like to have something integrated with another type of tools : autocomplete tools : #1623

**Comment by balex25** [comment]
Hey, I am interested in an Emoji Picker Plugin. So, 2023, does anyone know one?

**Comment by madromas** [comment]
Something like this? https://github.com/madromas/editorjs-emoji-tool
